web-server/v0.4/src/components/PageHeader/index.test.js

```diff
@@ -19,4 +37,72 @@ describe('test getBreadcrumb', () => {
     const urlNameList = urlToList('/search').map(url => getBreadcrumb(routerData, url).name);
     expect(urlNameList).toEqual(['search']);
   });
+
+  it('check getBreadcrumb function implementation', () => {
+    const breadcrumbNameMap = { url: { url: 'urlItem' } };
+    const url = 'url';
+    const breadcrumb = breadcrumbNameMap[url];
+    expect(getBreadcrumb(breadcrumbNameMap, url)).toBe(breadcrumb);
+  });
+});
+
+describe('test rendering of TableFilterSelection page component', () => {
+  it('render with empty props', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('Check functions implementation', () => {
+    const wrapper2 = shallow(<PageHeader {...mockProps} />);
+    const getBreadcrumbDom = jest.spyOn(wrapper.instance(), 'getBreadcrumbDom');
+    const conversionBreadcrumbList = jest.spyOn(wrapper.instance(), 'conversionBreadcrumbList');
+    const getBreadcrumbProps = jest.spyOn(wrapper.instance(), 'getBreadcrumbProps');
+    const conversionFromLocation = jest.spyOn(wrapper.instance(), 'getBreadcrumbProps');
+
+    wrapper2.setProps({ selectedControllers: ['a', 'b', 'c'] });
+    getBreadcrumbDom();
+    expect(wrapper.state('breadcrumb')).toEqual(conversionBreadcrumbList());
+    expect(conversionBreadcrumbList).toHaveBeenCalled();
+
+    expect(getBreadcrumbProps).toHaveBeenCalled();
+
+    // eslint-disable-next-line no-unused-vars
```

Comment:
We can omit the unused variables from the destructured reference and remove the disabled eslint line here to improve readability.